docs/Correctly-Using-Regular-Expressions.md

```diff
@@ -60,7 +60,7 @@ Platform
    </td>
    <td>“^” or “\A”
    </td>
-   <td>“\z”; “$” works but some documents conflict
+   <td>“\z”; [“$” works but some documents conflict](./Correctly-Using-Regular-Expressions-Rationale#java)
```

Comment:
It seems that this link does not render correctly because it is within a HTML table cell. Why is this a HTML table in the first place?